x-pack/plugins/index_lifecycle_management/__jest__/components/edit_policy.test.js

```diff
@@ -97,6 +97,11 @@ const setPhaseAfter = (rendered, phase, after) => {
   afterInput.simulate('change', { target: { value: after } });
   rendered.update();
 };
+const setPhaseIndexPriority = (rendered, phase, priority) => {
+  const priorityInput = rendered.find(`input#${phase}-phaseIndexPriority`);
```

Comment:
Can we use a `data-test-subj` instead, to uncouple tests from the app logic that uses the `id`?